libcore/marlin-file-operations.c

```diff
@@ -2360,14 +2039,26 @@ get_unique_target_file (GFile *src,
 
     max_length = pf_file_utils_get_max_name_length (dest_dir);
 
-    info = g_file_query_info (src,
-                              G_FILE_ATTRIBUTE_STANDARD_EDIT_NAME,
-                              0, NULL, NULL);
+    info = g_file_query_info (
+        src,
+        g_strjoin (
+            ",",
+            G_FILE_ATTRIBUTE_STANDARD_EDIT_NAME,
+            G_FILE_ATTRIBUTE_STANDARD_TYPE,
+            NULL
+        ),
```

Comment:
that's a leak, you can use the const way here:
```suggestion
        G_FILE_ATTRIBUTE_STANDARD_EDIT_NAME "," 
        G_FILE_ATTRIBUTE_STANDARD_TYPE,
```